java/src/main/java/com/amido/stacks/workloads/menu/service/v1/MenuService.java

```diff
@@ -0,0 +1,51 @@
+package com.amido.stacks.workloads.menu.service.v1;
+
+import com.amido.stacks.workloads.menu.commands.CreateMenuCommand;
+import com.amido.stacks.workloads.menu.commands.UpdateMenuCommand;
+import com.amido.stacks.workloads.menu.domain.Menu;
+import com.amido.stacks.workloads.menu.exception.MenuAlreadyExistsException;
+import com.amido.stacks.workloads.menu.repository.MenuRepository;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MenuService {
+
+  protected MenuRepository menuRepository;
+
+  public Optional<Menu> create(Menu menu) {
```

Comment:
When I create something I want a Menu back, not an Optional<Menu>
Using Optional places a burden on the caller

Reply:
return Optional.of(menu);